src/utils/taskmanager.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
This is a really good taskmanager file. I could see other EarthEngine users wanting to use this.

Reply:
Yes, it is especially useful when when you don't want to submit hundreds of tasks to EE, when even cancelling them can take dozen minutes.